src/build/cudos-blast.md

```diff
@@ -53,8 +53,8 @@ Cudos Blast can be used through a local installation in your project or by insta
 Create an npm project by going to an empty folder, then run
 `npm init`
  and follow the instructions. Once your project is ready, run 
-`npm install cudos-blast`
- To use your local installation of Cudos Blast, use `npx blast`
+`npm install cudos-blast`.
```

Comment:
having don'ts only here will make the documentation on consistent

Reply:
Linguistically this sentence was wrong. You can't start a new sentence without terminating the last one with a dot.

Reply:
I think we need the punctuation because the quoted commands are displayed on the same line as the description text.